server/storage.lua

```diff
@@ -164,7 +209,7 @@ function FetchPlayerEntity(citizenId)
         charinfo = json.decode(player.charinfo),
         job = player.job and json.decode(player.job),
         gang = player.gang and json.decode(player.gang),
-        position = json.decode(player.position),
+        position = vec4(position.x or QBCore.Config.DefaultSpawn.x, position.y or QBCore.Config.DefaultSpawn.y, position.z or QBCore.Config.DefaultSpawn.z, position.w or QBCore.Config.DefaultSpawn.w),
```

Comment:
Consider extracting a common function to use by FetchPlayerEntity and FetchPlayerEntities.